R/util.R

```diff
@@ -632,3 +632,19 @@ saverds_atomic <- function(data, path, allow_fail = FALSE) {
       finally = unlink(tmp))
   }
 }
+
+
+paths_are_different <- function(x, y) {
+  tryCatch({
+    x_real <- fs::path_real(x)
```

Comment:
Any reason not to use `path_norm()` and not have it fail here?

Reply:
I thought maybe these could be symbolic links, and we'd want to be able to tell they are the same path. On the other hand there are probably 0 users that have symbolic links in their orderly workspaces that it is probably overkill.